packages/SketchMorph2-Core.package/M2Backend.class/instance/project.at.rule..st

```diff
@@ -1,10 +1,10 @@
 drawing
-project: aForm at: aPoint
+project: aForm at: aPoint rule: aCombinationRule
```

Comment:
Maybe we should still provide a `project:at:` method that calls `project:at:rule:` with `Form blend` as rule. This would allow us to not change every caller of this method.